app/templates/events.hbs

```diff
@@ -1 +1,16 @@
+<h1>Manage events</h1>
+<div class="ui fluid pointing secondary menu">
```

Comment:
Put this also inside a grid. So that the width aligns with the width of the table. (These small details are important too)